script/upgrade/UpgradeGroupingWorkflows.s.sol

```diff
@@ -52,5 +52,8 @@ contract UpgradeGroupingWorkflows is UpgradeHelper {
         _writeAddress("RoyaltyWorkflows", address(royaltyWorkflows));
         _writeAddress("SPGNFTBeacon", address(spgNftBeacon));
         _writeAddress("SPGNFTImpl", address(spgNftImpl));
+        _writeAddress("DefaultStoryNftTemplate", defaultStoryNftTemplateAddr);
+        _writeAddress("OrgNFT", orgNftAddr);
+        _writeAddress("StoryNFTFactory", storyNftFactoryAddr);
```

Comment:
Why do we also need to write the address of Story NFT when upgrade GroupinWorkflow?

Reply:
Resolved in [9b71ffc](https://github.com/storyprotocol/protocol-periphery-v1/pull/92/commits/9b71ffc1d8b47f581ac487086f36f3dab929b9c5) :)